migrate_config.py

```diff
@@ -0,0 +1,44 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+
+from __future__ import unicode_literals
+
+import json
+import sys
+
+with open(sys.argv[1]) as fixture_file:
+    fixture = json.load(fixture_file)
+
+events = {}
+
+for obj in fixture:
+    if obj['model'] == 'django_emarsys.event':
+        events[obj['pk']] = obj
+        obj['params'] = {}
+
+for obj in fixture:
+    if obj['model'] == 'django_emarsys.eventdata':
+        event_obj = events[obj['fields']['event']]
+        event_obj['params'][obj['fields']['kwarg_name']] = obj['fields']
+
+event_config = {}
+
+for unused_event_id, event_obj in events.items():
+    print event_obj
+    event_config[event_obj['fields']['name']] = {
+        arg: (params['name'], '.'.join(params['content_type']))
+        for arg, params in event_obj['params'].items()
+    }
+
+print "EMARSYS_EVENTS = {"
+for event_name, params in sorted(event_config.items(),
+                                 lambda a, b: cmp(a[0], b[0])):
+    print u'    "{}": {{'.format(event_name.replace('"', '\"'))
```

Comment:
```python
In [1]: print repr("Ich sagte: \"Du sagtest: 'Öh, doch!'\"")
'Ich sagte: "Du sagtest: \'\xc3\x96h, doch!\'"'

In [2]: print '"{}"'.format("Ich sagte: \"Du sagtest: 'Öh, doch!'\"".replace('"', '\\"'))
"Ich sagte: \"Du sagtest: 'Öh, doch!'\""
```
Of course I considered `repr()`, but I didn't want the utf-8 byte representation there. Also keep in mind that this code won't even make it into the end product. It's just a tool to generate the settings.

But this made me realize that `'\"'` should have been `'\\"'`.